lib/reporters/list.js

```diff
@@ -8,6 +8,12 @@
 
 var Base = require('./base');
 var inherits = require('../utils').inherits;
+var constants = require('../runner').constants;
+var RUNNER_EVENT_PASS = constants.RUNNER_EVENT_PASS;
+var RUNNER_EVENT_FAIL = constants.RUNNER_EVENT_FAIL;
+var RUNNER_EVENT_BEGIN = constants.RUNNER_EVENT_BEGIN;
```

Comment:
Still dislike many of these constant names. For example, think the four root suite-related events should express that in their constant names (e.g., `RUNNER_EVENT_BEGIN` vs. `EVENT_ROOT_SUITE_BEGIN`. Same for test events (e.g., `RUNNER_EVENT_PASS` vs. `EVENT_TEST_PASS`). Both the `RUNNER_` and `SUITE_` prefixes don't add anything (given the `require`) [IMO], making the names unnecessarily longer without providing additional context.

Reply:
I want to ensure the constant names are all unique.  This will help code completion, and if you need to use two similarly-named constants from two different modules.  For example, `SUITE_BEGIN` could be a problem, as currently written.

I'll go back through them and ensure all the constant names are unique and see if it makes sense to remove the prefixes.

- [ ] audit names; attempt to remove prefixes

Reply:
Also, I want to ensure each module has its own constants.  If the event names are the same strings in two different modules, we shouldn't conflate these into a single constant.

Reply:
I've done the best I can with the names.

Reply:
Eh, I have a couple more to change...